worker_health_safety_usage_demo.ipynb

```diff
@@ -0,0 +1,307 @@
+      "text/plain": [
+       "<Figure size 720x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "res = visualize(images[0], predictions[0], 0.5)\n",
```

Comment:
I would show every image, otherwise why should we run 10 inferences to show just a single image?